src/js/providers/html5.js

```diff
@@ -87,7 +87,7 @@ define([
 
                 //play: _onPlayHandler, // play is attempted, but hasn't necessarily started
                 //loadstart: _generalHandler,
-                //loadeddata: _onLoadedData, // we have duration
+                loadeddata: _onLoadedData, // we have duration
```

Comment:
Can we update the comment for this event. What are we really looking for on this event? Tracks for one. Anything else?